### PR TITLE
Review of my trial project

### PR DESCRIPTION
**Pull request by christianbiring1**
In this exercise, I have to copy the code for one of the trials' projects and review it by: 

- [ ]  Apply best practice in HTML
- [ ] use semantic HTML tags 
- [ ] use CSS selector correctly.
- [ ] use CSS box model.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Review | trial</title>
+    <link rel="stylesheet" href="style.css" />
```
Comment (christianbiring1) [inline index.html:6]: kindly add the meta tags in the head of your HTML file! as view-port and the other missing.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <div id="img-div">
+        <img src="assets/download.jpeg" alt="" id="image" /><br />
+        <div id="img-caption">
+          born 28 December 1969) is a Finnish-American software engineer who is
+          the creator and, historically, the main developer of the Linux kernel,
+          used by Linux distributions and other operating systems such as
+          Android. He also created the distributed version control system Git
+          and the scuba dive logging and planning software Subsurface.
+        </div>
+      </div>
```
Comment (christianbiring1) [inline index.html:21]: Kindly 

- [ ] wrap your image within  a <figure> tag.
- [ ] wrap the text either with a <p> tag or put it inside a <figcaption> tag for describing what your image id for.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      <article id="tribute-info">
+        If you have time, you should read more about this incredible human being
+        <a
+          id="tribute-link"
+          href="https://en.wikipedia.org/wiki/Linus_Torvalds"
+          target="_blank"
+          >on his Wikipedia entry.</a
+        >
+      </article>
```
Comment (christianbiring1) [inline index.html:30]: kindly Change your tag <article> is not a proper semantic tag here.

**Review by christianbiring1 — COMMENTED** [review]
Great for making this project so far!! keep doing the great work :clinking_glasses: 
#Changes required :recycle: 
Please check my comment below and make all the required changes.
Kindly make sure that your project passes all the linters test.

File: index.html (hunk 2 above)
Comment (zainsadaqat) [reply index.html:21]: Good Job 👏 @christianbiring1 for explaining what changes are required and why you suggesting a change. Well done! 😄

File: index.html (hunk 1 above)
Comment (zainsadaqat) [reply index.html:6]: Good Job 👏 @christianbiring1 for explaining what changes are required and why you suggesting a change. Well done! 😄

File: index.html (hunk 3 above)
Comment (zainsadaqat) [reply index.html:30]: Great Job, you may tell them what tags can be used. 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hello @christianbiring1 😃 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 
# 👏🏻 👏🏻 👏🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  Descriptive Pull Request

## Required Changes ♻️

You may have noticed, there're some linters that are failing and you didn't ask the student to fix this. Kinldy ask them to fix the linters. 😄 

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (christianbiring1) [reply index.html:30]: ooh! My bad. when I was typing the message I have enclosed the tags in <  that's why they didn't show up on my comment. 
Here it is not. I edit my comment already.
Kindly change your tag <article and use a paragraph <p.

**Comment by christianbiring1** [comment]
Yeah yeah! of course, I did mention the linters errors right on my main comment just at the beginning of the review if you may check well. before I mention the errors about meta tags in the head.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @christianbiring1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**